Tests/SonarQube.TeamBuild.PreProcessor.Tests/PreProcessorExeTests.cs

```diff
@@ -76,17 +72,6 @@ private static void CheckExecutionFails(params string[] args)
             Assert.IsFalse(string.IsNullOrWhiteSpace(errorOutput), "Expecting error output if the process fails");
         }
 
-        private static void CheckExecutionSucceeds(params string[] args)
-        {
-            Process p = Execute(args);
-
-            Assert.AreEqual(0, p.ExitCode, "Expecting a zero exit code");
-            string errorOutput = p.StandardError.ReadToEnd();
-            Console.WriteLine("Error output: {0}", errorOutput);
-
-            Assert.IsTrue(string.IsNullOrWhiteSpace(errorOutput), "Not expecting error output if the process succeeds");
-        }
-
```

Comment:
Dead code. We can't easily shell the pre-processor exe and have it run successfully to completion as it calls out to a SonarQube server.